CalibPPS/TimingCalibration/plugins/PPSDiamondSampicTimingCalibrationPCLHarvester.cc

```diff
@@ -275,7 +275,7 @@ void PPSDiamondSampicTimingCalibrationPCLHarvester::dqmEndJob(DQMStore::IBooker&
 
 void PPSDiamondSampicTimingCalibrationPCLHarvester::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
-  desc.add<std::string>("timingCalibrationTag", "GlobalTag:DiamondSampicCalibration")
+  desc.add<std::string>("timingCalibrationTag", "GlobalTag:PPSDiamondSampicCalibration")
```

Comment:
do you really need to specify this?
I'd just do
```
timingCalibrationToken_ = esConsumes<edm::Transition::BeginRun>();
```
and then get rid of `timingCalibrationTag`, let the GT deal with the rest

Reply:
@tvami we have multiple tags with the same record type in the GT. Correct me if I am wrong but in such case one needs to specify the name to differentiate between the two, right?

Reply:
In that case one should specify the label, not the name of the tag (which does change time to time -- unlike the label, that is fixed)

Reply:
@tvami but this is a label: https://cms-conddb.cern.ch/cmsDbBrowser/search/Prod/PPSDiamondSampicCalibration
Unless you're talking about something else.

Reply:
I guess it depends on if you treat a label as part of the tag or not. Because this is really GlobalTag+label, right? Would you like this to be renamed?

Reply:
> Would you like this to be renamed?

Yes, but it can be in another PR

Reply:
Would using `edm::ESInputTag` directly as the configuration parameter type help to reduce the confusion?

Reply:
Well, let's do this: if due to https://github.com/cms-sw/cmssw/pull/38893#issuecomment-1199193514 you have to modify this PR, then please change the name. OK?